python/pythondotorg
# feat(frontend): update sass compiler/toolchain

This updates us to the latest dart based implementation of sass,

- Vendors compass/susy which used to be provided via ruby installation
- patches susy to comply with dart-sass@1.30.0's expectations for line continuation
- updates our scss files to comply with dart-sass@1.30.0's expectations
- add a check to ensure compiled SCSS matches committed CSS
- move to GitHub Actions for CI
- Add a Makefile with `runserver`, `sass`, and `sass-watch` targets.

Notes:

Used dart implementation of Sass from https://github.com/sass/dart-sass/releases/tag/1.30.0
